packages/lib-classifier/src/store/TranscriptionReductions/TranscriptionReductions.spec.js

```diff
@@ -0,0 +1,88 @@
+import sinon from 'sinon'
+import { reducedEmptySubject, reducedSubject } from './mocks'
+
+import TranscriptionReductions, { caesarClient } from './TranscriptionReductions'
+
+describe('Models > TranscriptionReductions', function () {
+  let reductions
+
+  before(function () {
+    // sinon.stub(console, 'error')
+    reductions = TranscriptionReductions.create({
+      caesarReducerKey: 'ext',
+      subjectId: '13971150',
+      workflowId: '5339'
+    })
+  })
+
+  after(function () {
+    // console.error.restore()
+  })
+
+  it('should exist', function () {
+    expect(reductions).to.be.ok()
+  })
+
+  describe('with transcribed lines', function () {
+    before(async function () {
+      sinon.stub(caesarClient, 'request').callsFake(() => Promise.resolve(reducedSubject))
+      reductions = TranscriptionReductions.create({
+        caesarReducerKey: 'ext',
+        subjectId: '13971150',
+        workflowId: '5339'
+      })
+      await reductions.fetchCaesarReductions()
+    })
+
+    after(function () {
+      caesarClient.request.restore()
+    })
+
+    it('should default to frame 1', function () {
```

Comment:
I think this should actually default to 0. It just happened that the first ASM subject I looked at only had reductions for `frame1` and `frame2`, so I assumed the numbering started at 1.